lib/poolify.opt.js

```diff
@@ -12,15 +12,15 @@ const poolify = (factory, min, norm, max) => {
   let allocated = norm;
   const items = duplicate(factory, norm);
   const delayed = [];
-  return (par) => {
+  return par => {
     if (Array.isArray(par)) {
       while (par.length) {
         const item = par.shift();
         const request = delayed.shift();
         if (request) request(item);
         else items.push(item);
       }
-      return;
+      return undefined;
```

Comment:
`return pool;`

Reply:
@tshemsedinov there is no `pool` available in this scope.

Reply:
We need to put `par => {` to identifier `pool` and return it